multiapi-engine/src/main/java/com/sngular/api/generator/plugin/openapi/utils/MapperContentUtil.java

```diff
@@ -29,48 +29,62 @@
 import org.apache.commons.lang3.ObjectUtils;
 import org.apache.commons.lang3.StringUtils;
 
-public class MapperContentUtil {
-
-  private static final String OBJECT = "Object";
-
-  private static final Set<String> BASIC_TYPES = Set.of("String", "Integer", OBJECT);
+import static com.sngular.api.generator.plugin.openapi.model.SchemaFieldObjectType.ARRAY;
+import static com.sngular.api.generator.plugin.openapi.model.SchemaFieldObjectType.MAP;
+import static com.sngular.api.generator.plugin.openapi.model.SchemaFieldObjectType.BIG_DECIMAL;
+import static com.sngular.api.generator.plugin.openapi.model.SchemaFieldObjectType.OBJECT;
+import static com.sngular.api.generator.plugin.openapi.model.SchemaFieldObjectType.ENUM;
+import static com.sngular.api.generator.plugin.openapi.model.SchemaFieldObjectType.INTEGER;
+import static com.sngular.api.generator.plugin.openapi.model.SchemaFieldObjectType.LONG;
+import static com.sngular.api.generator.plugin.openapi.model.SchemaFieldObjectType.FLOAT;
+import static com.sngular.api.generator.plugin.openapi.model.SchemaFieldObjectType.DOUBLE;
+import static com.sngular.api.generator.plugin.openapi.model.SchemaFieldObjectType.STRING;
+import static com.sngular.api.generator.plugin.openapi.model.SchemaFieldObjectType.BASIC_TYPES;
 
-  private static final String ARRAY = "array";
-
-  private static final String MAP = "map";
+public class MapperContentUtil {
 
-  private static final String BIG_DECIMAL = "bigDecimal";
+  private static final String ADDITIONAL_PROPERTY_NAME = "AdditionalProperty";
 
-  private static final String INTEGER = "integer";
+  private MapperContentUtil() {
+  }
 
-  private static final String DOUBLE = "double";
+  public static Map<String, SchemaObject> mapComponentToSchemaObject(
+      final Map<String, Schema> totalSchemas, final Schema<?> schema, final String nameSchema, final SpecFile specFile) {
+    return mapComponentToSchemaObject(totalSchemas, new HashMap<>(), new ArrayList<>(), schema, StringUtils.defaultIfBlank(schema.getName(), nameSchema), specFile,
+                                      specFile.getModelPackage());
+  }
 
-  private static final String FLOAT = "float";
+  private static Map<String, SchemaObject> mapComponentToSchemaObject(
+      final Map<String, Schema> totalSchemas, final Map<String, SchemaObject> compositedSchemas, final List<String> antiLoopList, final Schema<?> schema, final String nameSchema,
+      final SpecFile specFile, final String modelPackage) {
+    antiLoopList.add(nameSchema);
 
-  private static final String LONG = "long";
+    if (Objects.isNull(schema.getName())) {
+      schema.setName(nameSchema);
+    }
+    final var listSchema = getFields(totalSchemas, schema, specFile, compositedSchemas, antiLoopList);
 
-  private static final String STRING = "string";
+    String schemaCombinatorType = "";
+    if (Objects.nonNull(schema.getAllOf())) {
+      schemaCombinatorType = "allOf";
```

Comment:
I think you should create an static variable instead of "hardcoding" these values.